test/plugins/validation/2and3/operation-ids.js

```diff
@@ -164,14 +171,18 @@ describe('validation plugin - semantic - operation-ids', function() {
     };
 
     const resolvedSpec = await resolver.dereference(spec);
-    const res = validate({ resolvedSpec });
+    const res = validate({ resolvedSpec }, config);
 
     expect(res.errors.length).toEqual(0);
     expect(res.warnings.length).toEqual(12);
     expect(res.warnings[0].path).toEqual('paths./books.get.operationId');
     expect(res.warnings[0].message).toEqual(
       'operationIds should follow consistent naming convention. operationId verb should be list'
     );
+
+    const customRes = validate({ resolvedSpec }, customConfig);
+    expect(customRes.errors.length).toEqual(0);
+    expect(customRes.warnings.length).toEqual(0);
```

Comment:
I appreciate the thought here, but these added checks are unnecessary. We can assume the message carrier logic works as it has it's own tests.

Also, just for future reference, it is best practice to test different behaviors in entirely different tests (different `it` functions), rather than adding checks for multiple behaviors within the same test.